src/applications/vaos/appointment-list/sass/styles.scss

```diff
@@ -143,6 +143,13 @@
   }
 }
 
+.vaos-appts__breadcrumb li > button[aria-current="page"] {
+  cursor: default;
+  pointer-events: none;
+  text-decoration: none;
+  color: $color-black !important;
+}
```

Comment:
May want to add "font-weigh: bold;" based on design spec for the non linkable nav text